danger-klaxit/lib/klaxit/config_sorter.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+module Klaxit
+  class ConfigSorter
+    attr_reader :file
+
+    # Regexp that detects a group, and returns its name as first match.
+    PARENT_RE = /\A(\w+):/.freeze
+    # Regexp that detects any child key in a group, and returns its name as
+    # first match.
+    CHILD_RE = /\A  (\w+):/.freeze
+
+    def initialize(file)
+      @file = file
+    end
+
+    def sorted_file
+      @sorted_file ||= parse
+    end
+
+    # Check if there is a change in the +file+, in other words: check if the
+    # order was not correct.
+    def changed?
+      @changed ||= sorted_file != IO.read(file)
+    end
+
+    def diff
+      require "tempfile"
+      new_file = Tempfile.new(["new_config.", ".yml"])
```

Comment:
```suggestion
      new_file = Tempfile.new(["new_config", ".yml"])
```

Reply:
Nop, that is a choice to have a more readable path ;)

Reply:
I don't understand why ^^